src/databricks/labs/ucx/install.py

```diff
@@ -231,6 +231,9 @@ def _apply_upgrades(self):
         try:
             upgrades = Upgrades(self._product_info, self._installation)
             upgrades.apply(self._ws)
+        except InvalidParameterValue as err:
```

Comment:
Can be combined with the `NotFound` except below
```suggestion
        except (NotFound, InvalidParameterValue) as err:
```

https://docs.python.org/3/tutorial/errors.html#handling-exceptions